src/index.js

```diff
@@ -0,0 +1,151 @@
+"use strict";
+
+
+const state = {
+    // start temparture with info from API call
+    // state city name from
+    city: "Seattle",
+    temperature: 0, 
+    lat: 0,
+    lon: 0,
+    original_landscape: "🌵🌼🌸🍁🌵🌼🌸🍁"
+};
+
+const getCityName = (event) => {
+    const inputElement = document.getElementById("city-name")
+    state.city = inputElement.value;
+    const cityDisplay = document.querySelector("h3");
+    cityDisplay.textContent = "For the lovely city of " + "🌟" + state.city + "🌟";
+}
+
+const increaseTemp = (event) => {
+    state.temperature ++;
+    const tempCount = document.getElementById("temp-count");
+    tempCount.textContent = `${state.temperature}`;  
+    changeColorByTemp();
```

Comment:
Since this bit of code is repeated again in L30-32, it might be a good idea to pull it out into its own helper function!